src/health.h

```diff
@@ -280,55 +287,57 @@ typedef struct rrdcalctemplate {
 #define HEALTH_ENTRY_FLAG_EXEC_FAILED  0x00000008
 #define HEALTH_ENTRY_FLAG_SAVED        0x10000000
 
+/** List of alarm_entry */
 typedef struct alarm_entry {
-    uint32_t unique_id;
-    uint32_t alarm_id;
-    uint32_t alarm_event_id;
+    uint32_t unique_id;      ///< Uniqe id.
+    uint32_t alarm_id;       ///< id of alarm
+    uint32_t alarm_event_id; ///< id of event
 
-    time_t when;
-    time_t duration;
-    time_t non_clear_duration;
+    time_t when;               ///< timestamp when occoured
+    time_t duration;           ///< duration time
+    time_t non_clear_duration; ///< approximate duration time
 
-    char *name;
-    uint32_t hash_name;
+    char *name;         ///< the name
+    uint32_t hash_name; ///< hash of `name`
 
-    char *chart;
-    uint32_t hash_chart;
+    char *chart;         ///< name of chart
+    uint32_t hash_chart; ///< hash of chart
 
-    char *family;
+    char *family; ///< link to family
 
-    char *exec;
-    char *recipient;
-    time_t exec_run_timestamp;
-    int exec_code;
+    char *exec;                ///< ktsaou: your help needed
+    char *recipient;           ///< ktsaou: your help needed
+    time_t exec_run_timestamp; ///< timestamp when `exec` run
+    int exec_code;             ///< exit code of `exec`
 
-    char *source;
-    char *units;
-    char *info;
+    char *source; ///< ktsaou: your help needed
```

Comment:
the source description of the alarm that generated this alarm (i.e. `LINE@FILE`).